### ISSUE TITLE
[Screen Reader – Intelligent Support - IRIS]: The role of 'Continue' element is announced incorrectly as a toggle button

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4611]
### **PWD Impact:**
Users relying on screen reader will be affected while trying to navigate through the page if the role of elements are announced incorrectly.

### **Repro Steps:**
Step 1: Open the above URL in a browser. Home Page will open. 
Step 2: Navigate to Iris bot icon in the header and invoke it
Step 3: Iris bot chat window is displayed
Step 4: Type start over-> Top questions for my role-> select Where can I download the Time 2.0 mobile app for Android ->Select no create a support ticket->select any tool from the drop down->continue button
Step 5: After invoking the continue button a form is displayed, turn on Narrator.
Step 6: Verify if when navigating to the 'Continue' button the role is announced as toggle button.
### **Actual result:**
While using narrator if we check the role of the 'Continue' button it is announced as toggle button off and if we click on it it will announce as toggle button on. It can be seen visually as well that the role announced is incorrect.
### **Expected Result:**
The role should be that of a button, the role of toggle button is incorrect. 
PFA<MAS 4.1.2 - Intelligent Support - IRIS - Narrator>
### **Note:** 
Similar behavior observed in NVDA.
PFA<MAS 4.1.2 - Intelligent Support - IRIS - NVDA>
### **MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7d&file=MAS%204.1.2%20%E2%80%93%20Name%2c%20Role%2c%20Value.docx&action=default&mobileredirect=true&share=IQGQXXtQufHhSpHRLx8tbHwdAUoz-PLo6lZ2eagKqmDeOGc
### **Test Environment:**
OS: Windows 11 Enterprise version 21H2
Browser: New Edge Version 106.0.1370.42 (Official build) (64-bit)
URL: https://servicesuat.microsoft.com/#/
Screen Reader: Narrator

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS4.1.2;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Jul22;

**Comment by compulim** [comment]
I have been asking access for the few weeks and did not get any responses.

This could possibly a bug on Adaptive Cards. Their buttons is `<button type="button" aria-pressed="false">`.

Since I cannot investigate the Iris bot, I am closing this bug.

**Comment by pgopinadhu** [comment]
@compulim Product Team Provided the Environment access. Please check and let us know if there are any issues. Thanks!

**Comment by compulim** [comment]
I verified the Iris Bot is running Web Chat 4.15.7.

I am not able to repro the issue. In the recordings below, around 00:19, I can hear "Did that answer your question or solve your problem?"

https://user-images.githubusercontent.com/1622400/224416455-5b6cd992-df2c-44f4-9c43-e96637ecd9ee.mp4

**Comment by compulim** [comment]
This is an Adaptive Card issue, please file the bug to [`microsoft/adaptivecards`](https://github.com/microsoft/AdaptiveCards).

The attachment read "Continue, button, off" which means it is a push button.

The design of Adaptive Cards is that all cards can be submitted multiple times and repetitively. They are not one-off actions. Thus, in AC, all buttons are marked as `ac-pushButton`, even though their button is a submit button.

![image](https://user-images.githubusercontent.com/1622400/224422218-3d413474-1150-4b14-b63d-246c7d6c9302.png)

Please bring this issue to AC, as this is something related to their design/schema.

https://user-images.githubusercontent.com/1622400/224421451-35e7dd45-4189-4150-8bbb-73856b0314e3.mp4
